ignite/metrics.py

```diff
@@ -0,0 +1,86 @@
+import torch
+
+
+def _convert_history(history):
+    y_preds, ys = zip(*history)
+    y_pred = torch.cat(y_preds, dim=0)
+    y = torch.cat(ys, dim=0)
+    return y_pred, y
+
+
+def binary_accuracy(history):
```

Comment:
we should probably accept a transform function that takes the history / an item from the history and transforms it into the form `(y_pred, y)` by default this can just be `lambda x: x`